### ISSUE TITLE
NVDA Audio ducking and Touch Screen does not working

### ISSUE THREAD

**Issue by borfirbora** [nvaccess/nvda#14736]
### Actual behavior:

I just bought my windows 10 computer. For the first few days, my Touchscreen and sound attenuation were working fine. My computer recently got the Windows 11 update. Then my touchscreen and audio attenuation feature are not supported.

### Expected behavior:

Normally it should say the item when I tap the screen, it should activate when I double click it.

### NVDA logs, crash dumps and other attachments:

DEBUGWARNING - touchHandler.touchSupported (10:12:32.567) - MainThread (5236):
NVDA doesn't have UI Access so touch isn't supported.
INFO - core.main (10:12:33.044) - MainThread (5236):
NVDA initialized
DEBUGWARNING - characterProcessing._getSpeechSymbolsForLocale (10:12:33.130) - MainThread (5236):
No CLDR data for locale en_US
DEBUGWARNING - displayModel.DisplayModelTextInfo._get__storyFieldsAndRects (10:12:38.528) - MainThread (5236):
AppModule does not have a binding handle

#### NVDA installed/portable/running from source:

NVDA installed.

#### NVDA version:

2022.4.0.27401

#### Windows version:

Version 22H2 (22621.1413)

#### Name and version of other software in use when reproducing the issue:

All Applications.

#### Other information about your system:

* Model: HP ENVY x360 2-in-1 Laptop 13-bf0xxx
* No: 6Z5W1EA#AB8
* Processor: 12th Gen Intel(R) Core(TM) i5-1230U

**Comment by borfirbora** [comment]
The interesting thing is
  When I first opened it, I was using it without any problems. Whatever happened, it happened after the update of Windows Update.

**Comment by borfirbora** [comment]
I tried everything you said. The result is unfortunately the same.

29 Mart 2023 Çarşamba tarihinde James Teh ***@***.***> yazdı:

**Comment by borfirbora** [comment]
This digital signature is okay

29 Mart 2023 Çarşamba tarihinde James Teh ***@***.***> yazdı:

**Comment by jcsteh** [comment]
This is very odd. NVDA can't get the uiAccess privilege on your system. That should only happen if nvda.exe isn't in a trusted system directory (such as Program Files) or if the signature isn't valid. But based on the info you've provided, neither of these things is true. I can't see any [Windows policy or registry setting related to uiAccess](https://learn.microsoft.com/en-us/windows/security/identity-protection/user-account-control/user-account-control-group-policy-and-registry-key-settings) that should prevent uiAccess otherwise.

**Comment by borfirbora** [comment]
Yes! Turning on UAC fixed my audio ducking and touchscreen. Isn't there a solution to this?

**Comment by lukaszgo1** [comment]
This problem is tracked in #13591, so this issue should be closed as a duplicate.
